extensions/ql-vscode/src/skeleton-query.ts

```diff
@@ -91,7 +91,15 @@ export class SkeletonQueryWizard {
 
     const firstFolder = workspaceFolders[0];
 
-    return firstFolder.uri.path;
+    // For the vscode-codeql-starter repo, the first folder will be a ql pack
+    // so we need to get the parent folder
+    if (firstFolder.uri.path.includes("codeql-custom-queries")) {
```

Comment:
Can we make this work even better by doing this only after the language has been selected? It seems to me that the distinction between `folderName` and `storagePath` can be removed completely and we don't need this logic at all.

It should be possible to just have a single instance variable specifying the directory where the qlpack will be generated. For the codespace, this would be `<workspace>/codeql-custom-queries-${language}`. For the starter workspace, it would be the workspace directory path which matches the `codeql-custom-queries-${language}` name.

Reply:
I've moved the call to this method after the language selection and added some tests for it.